### DISCUSSION TITLE
multiple accounts setup - update interval

### DISCUSSION THREAD

**Discussion by pixbalance** [YorkshireIoT/ha-google-fit#131] [Q&A]
I want to add two accounts from two different google accounts. I have added the OAuth-Client-ID from account one and could add both accounts.
Account one is updating the numbers just once a day, account two is update a bit more often but also not according to the 5 min pull interval.
Any idea what I did wrong?

![image](https://github.com/YorkshireIoT/ha-google-fit/assets/62733848/b00b2955-d16a-4351-bc7d-d6c26cc04d33)

**Comment by YorkshireIoT** [comment]
If you enable debug logging for the integration it should tell you if it's found any data for the account in the logs, and if the integration is behaving correctly that should be every 5 minutes (or whatever you configured your polling interval to be).

If the integration is polling every 5 minutes and it's finding no data, it's most likely the problem is having earlier in the chain. Common issues can be data won't upload to the Google servers unless the phone is on WiFi, and also most modern phones now implement power savings for apps where they'll kill them if they haven't been used in a while. Depending on the phone, there is usually a way to get around this and make sure the app isn't put to sleep
